### DISCUSSION TITLE
Is whisper free?

### DISCUSSION THREAD

**Discussion by kevinco26** [openai/whisper#1285] [Q&A] [answered]
Hi! this may be a naive question but I can't seem to figure out the answer.

I see pricing on Openai's website: https://openai.com/pricing
**Whisper | $0.006 / minute (rounded to the nearest second)**

Then their examples involve using an authorization key in order to send the request.

However I've been using the python pip package and doing a bunch of tests on some audio by using both `transcribe()` method and the `log_mel_spectrogram()`

Interestingly I'm able to get results just fine and I'm not setting any openai api key anywhere... How is this working and where does the pricing come in?

**Comment by oakieoaktree** [comment, marked as answer]
The use of the models / python package as you are describing is free under the license indicated by the github project page (MIT).
Only the use of the hosted solution / service provided by OpenAI via the web API costs money.

**Reply by kevinco26** [reply]
thanks!

**Reply by hissain** [reply]
@oakieoaktree  does it mean, I can use that model commercially for free by hosting on my own machine?

**Reply by Purfview** [reply]
> @oakieoaktree does it mean, I can use that model commercially for free by hosting on my own machine?

Yes.
